spec/student_spec.rb

```diff
@@ -0,0 +1,24 @@
+require_relative '../student'
+require_relative '../person'
+
+describe Student do
```

Comment:
- [ ] Since we are told to test all methods in all classes, let's kindly add tests for `add_classroom` method in the Student class

Reply:
We added test for the add_classroom method...😊 Thanks!